server/modules/selva/module/selva_object.h

```diff
@@ -21,6 +21,12 @@ enum SelvaObjectType {
     SELVA_OBJECT_POINTER = 7,
 };
 
+enum SelvaObjectMetaSubtype {
+    SELVA_OBJECT_META_SUBTYPE_OBJECT = 0,
+    SELVA_OBJECT_META_SUBTYPE_RECORD = 1,
+    SELVA_OBJECT_META_SUBTYPE_TEXT = 2,
+};
```

Comment:
I would have put this into `include/selva.h` as `#define`s but this is fine too. `#define`s because if we later want to use other bits as flags then `enum` feels a bit dirty.

Reply:
Yeah I think define is better. I'll change that.